com.unity.netcode.gameobjects/Runtime/NetworkVariable/ResizableBitVector.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+
+namespace Unity.Netcode
+{
+    internal struct ResizableBitVector : INetworkSerializable, IDisposable
```

Comment:
Quick comment on what it does and possibly uses for this? (I remember the conversation we had still, but six+ months for now some general description could be useful)